Libraries/Components/WebView/WebView.ios.js

```diff
@@ -334,6 +334,21 @@ class WebView extends React.Component {
      * to tap them before they start playing. The default value is `true`.
      */
     mediaPlaybackRequiresUserAction: PropTypes.bool,
+
+    /**
+     * Rules that will be used to block the webview navigation
+     */
+    navigationBlockingPolicies: PropTypes.arrayOf(PropTypes.shape({
+      currentURL: PropTypes.string,
+      navigationType: PropTypes.string,
+      url: PropTypes.string,
```

Comment:
It seems like `navigationType` is only used in the iOS `RCTWebView.h`. That field isn't available in `ReactWebViewManager` on Android.

Though if we need `navigationType` as a policy at all (for blocking purposes) is unclear to me.